templates/circle-form.html

```diff
@@ -119,8 +119,8 @@ <h4 class="grid_separator circle-benefits__header">Questions?</h4>
         </div>
       </section>
 
-      <section class="circle-wall grid_separator--l">
-        <figure class="circle-wall__logo grid_separator--l">
+      <section class="wall grid_separator--l">
```

Comment:
The wall is half in HTML land, half in Vue land (my fault). Would it make sense to just have the entire thing in Vue, maybe allowing the component to accept a logo prop?

Reply:
yeah that bugged me too, but I didn't want to get too wild changing things. If you're cool with that, I'd prefer keeping it all in vue